nomination/templates/nomination/url_listing.html

```diff
@@ -161,14 +168,14 @@ <h4 class="panel-title">Metadata</h4>
                                         {% endfor %}
                                     {% elif met.form_type == "select" %}
                                         <p class="help-block">{{ met.help }}</p>
-                                        <select class="form-control" multiple size="6" id="{{ met.metadata.name }}" name="{{ met.metadata.name }}">
+                                        <select multiple size="6" id="{{ met.metadata.name }}" name="{{ met.metadata.name }}">
                                             {% for val in vals %}
                                                 <option value="{{ val.key }}">{{ val.value }}</option>
                                             {% endfor %}
                                         </select>
                                     {% elif met.form_type == "selectsingle" %}
                                         <p class="help-block">{{ met.help }}</p>
-                                        <select class="form-control" size="6" id="{{ met.metadata.name }}" name="{{ met.metadata.name }}">
+                                        <select size="17" id="{{ met.metadata.name }}" name="{{ met.metadata.name }}">
```

Comment:
Not sure if this `size="17"` is something you were intending to include.

Reply:
ha! No, not at all. I don't even know how that got in there! I must have hit "undo" one too many times at some point.